editor/editor_node.cpp

```diff
@@ -3020,6 +3001,40 @@ void EditorNode::_tool_menu_option(int p_idx) {
 	}
 }
 
+void EditorNode::_export_as_menu_option(int p_idx) {
+	if (p_idx == 0) { // MeshLibrary
+		current_menu_option = FILE_EXPORT_MESH_LIBRARY;
+
+		if (!editor_data.get_edited_scene_root()) {
+			show_accept(TTR("This operation can't be done without a scene."), TTR("OK"));
+			return;
+		}
+
+		List<String> extensions;
+		Ref<MeshLibrary> ml(memnew(MeshLibrary));
+		ResourceSaver::get_recognized_extensions(ml, &extensions);
+		file_export_lib->clear_filters();
+		for (const String &E : extensions) {
+			file_export_lib->add_filter("*." + E);
+		}
+
+		file_export_lib->popup_file_dialog();
+		file_export_lib->set_title(TTR("Export Mesh Library"));
+	} else {
```

Comment:
I'd add a comment to explain what the `else` case is used for. It's not immediately obvious to me that users would pass the callbacks for their plugin-added entries as metadata (makes sense, though I wonder if there isn't a "cleaner" way to do this, though I think this is likely outside the scope of this PR).

Reply:
Added the comment. Is it okay? Or should I explain it further?

Reply:
I would have put some more details about the "pass callbacks as item metadata" bit, but it's also fine as is I guess.